extensions/python/CMakeLists.txt

```diff
@@ -45,7 +45,15 @@ target_include_directories(minifi-python-script-extension PUBLIC ${Python_INCLUD
 
 get_target_property(TARGET_EXT minifi-python-script-extension SUFFIX)
 
-if (NOT WIN32)
+if (WIN32)
+    add_custom_command(
+        TARGET minifi-python-script-extension
+        POST_BUILD
+        COMMAND ${CMAKE_COMMAND} -E copy_directory
+            "${CMAKE_SOURCE_DIR}/extensions/python/pythonprocessors" "${CMAKE_BINARY_DIR}/pythonprocessors"
+        COMMENT "Copy example python processors to the build directory"
+    )
```

Comment:
There is a problem with attaching this post-build command to the python-script-extension target: if MiNiFi is built with `ENABLE_PYTHON=OFF`, then `cpack` will fail.  I am working on fixing this; depending on how long the fix takes, we can decide whether to merge this PR without the fix and do the fix later, or wait for the fix.

Reply:
fixed by a8102c59822f621eafa5009e18de5e9b303c9ff1